packages/paste-website/src/components/callout/index.tsx

```diff
@@ -4,13 +4,13 @@ import {StyledCallout} from './styles';
 import {CalloutProps, CalloutTitleProps, CalloutTextProps} from './types';
 
 const CalloutTitle: React.FC<CalloutTitleProps> = ({as, children}) => (
-  <Text as={as} marginBottom="space40" fontSize="fontSize20">
+  <Text as={as} marginBottom="space40" fontSize="fontSize30">
```

Comment:
if the default of `Text` is `fontSize="fontSize30"` you can probably remove these font size settings if the idea is to just get the default body text size. Otherwise, if we shift the scale again, you'll be changing these all over again.

Reply:
Makes total sense.